src/mutations/index.ts

```diff
@@ -88,6 +98,22 @@ export const MUTATIONS = {
       mutation: User.changeUserPassword,
       ...options,
     }),
+  RegisterAccount: <TCacheShape>(
+    client: ApolloClient<TCacheShape>,
+    options: MutationOptions<RegisterAccount, RegisterAccountVariables>
+  ) =>
+    client.mutate({
+      mutation: User.registerAccount,
+      ...options,
+    }),
+  ResetPasswordRequest: <TCacheShape>(
+      client: ApolloClient<TCacheShape>,
+      options: MutationOptions<ResetPasswordRequest, ResetPasswordRequestVariables>
+    ) =>
+      client.mutate({
+        mutation: User.resetPasswordRequest,
+        ...options,
+    }),
```

Comment:
I don't see it is used anywhere. This list of mutations was typically used in `APIProxy`, but we don't want to use it anymore, so it might be safely removed.